x-pack/plugins/timelines/public/components/t_grid/integrated/index.tsx

```diff
@@ -292,21 +290,22 @@ const TGridIntegratedComponent: React.FC<TGridIntegratedProps> = ({
       >
         {loading && <EuiProgress size="xs" position="absolute" color="accent" />}
 
+        {graphOverlay}
```

Comment:
THIS IS THE ACTUAL BUGFIX! [got deleted by accident](https://github.com/elastic/kibana/pull/108644/files#diff-5796dde9b49a93dcce15ee9ea2133002123cbb9802033e07ae9db0c9739d08c6L337)

Reply:
sorry about that